code/src/components/Actions.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { useDispatch, useSelector } from "react-redux";
+import styled from "styled-components";
+
+import { todos } from "../reducers/todos";
+import goodjob from "../assets/goodJob.png";
+
+const Buttons = styled.button`
```

Comment:
This is a button, so should be named `Button`

Reply:
fixed